shell/components/LocaleSelector.vue

```diff
@@ -12,6 +12,14 @@ export default {
       type:    String,
       default: ''
     },
+    tabbingIndex: {
+      type:    Number,
+      default: null,
+    },
```

Comment:
One more unused instance of `tabbingIndex` that can be removed.

Reply:
Don't know how I missed that. It's the last one lingering, I've made sure of that 😛 Fixed. Thanks